p4-16/psa/PSA.mdk

```diff
@@ -2135,7 +2135,7 @@ digest message can contain any from the dataplane.
 
 In PSA, digest is performed at the end of the ingress pipeline in the
 deparser. The digest extern provides a `pack` method to specify the
-content in the 
+content in the
```

Comment:
Might be good to add an explicit TBD here to remind ourselves of this incomplete sentence.  Better of course to actually complete it.